src/pages/HomePage/index.tsx

```diff
@@ -1,22 +1,18 @@
 import { Header } from "components/Layout/Header";
 import { PageLayout } from "components/Layout/PageLayout";
 import { ProductsTable } from "components/UI/Tables/ProductsTable";
+import { useProduct } from "hooks/useProduct";
 
-const HomePage = () => (
-  <PageLayout header={<Header />}>
-    <ProductsTable
-      products={[
-        {
-          id: "1",
-          name: "acd",
-          brand: "Adidas",
-          price: "123",
-          image:
-            "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png",
-        },
-      ]}
-    />
-  </PageLayout>
-);
+const HomePage = () => {
+  const { products, useFetchProducts } = useProduct();
+
+  useFetchProducts();
```

Comment:
```suggestion
```

Reply:
how should i do it? i returned this hook from useProduct and use it i need to fetch products

Reply:
U need to fetch the data on `useProduct`

Reply:
Fixed